apps/system-e2e/src/playwright.config.ts

```diff
@@ -1,11 +1,7 @@
-import type { PlaywrightTestConfig } from '@playwright/test'
-import { devices } from '@playwright/test'
-import './addons'
-/**
- * Read environment variables from file.
- * https://github.com/motdotla/dotenv
- */
-// require('dotenv').config();
+import { PlaywrightTestConfig } from '@playwright/test'
```

Comment:
I think this `type` thing was important for the bundler to work. Probably to best leave it be.